src/pages/home/ReportScreen.js

```diff
@@ -86,6 +91,8 @@ const defaultProps = {
     isComposerFullSize: false,
     betas: [],
     policies: {},
+    accountManagerReportID: {},
```

Comment:
The default value for `accountManagerReportID` should be the empty string. I am seeing its use to navigate to screen reports and using `ROUTES.getReportRoute` where the argument is a string.

Also, we have `this.props.accountManagerReportID && ReportUtils.isConciergeChatReport(this.props.report) && this.state.isBannerVisible` this condition to show the banner. If the default value is an object, the banner will show even if the prop is absent.